enhancing/enhancer.spec.js

```diff
@@ -1,2 +1,146 @@
+            };
+
+            // Act
+            const item = {
+                name: "broad sword",
+                durability: 100,
+                enhancement: 13
+            };
+
+            // Assert
+            expect(succeed(item)).toEqual(expected);
+        });
+
+        // test
+        it('item with an enhancement of 20 will remain the same', () => {
+
+            // Arrange
+            const expected = {
+                name: "broad sword",
+                durability: 100,
+                enhancement: 20
+            }
+
+            // Act
+            const item = {
+                name: "broad sword",
+                durability: 100,
+                enhancement: 20
+            };
+
+            // Assert
+            expect(succeed(item)).toEqual(expected)
+        });
+    });
+
+    // test suite
+    describe('the fail function', () => {
+
+        // test
+        it('will return item with -5 durability', () => {
+
+            // Arrange
+            const expected = {
+                name: "broad sword",
+                durability: 95,
+                enhancement: 11
+            };
+
+            // Act
+            const item = {
+                name: "broad sword",
+                durability: 100,
+                enhancement: 11
+            };
+
+            // Assert
+            expect(fail(item)).toEqual(expected);
+        });
+
+        // test
+        it('will return item with -10 durability', () => {
+            
+            // Arrange
+            const expected = {
+                name: "broad sword",
+                durability: 90,
+                enhancement: 15
+            };
+
+            // Act
+            const item = {
+                name: "broad sword",
+                durability: 100,
+                enhancement: 15
+            };
+
+            // Assert
+            expect(fail(item)).toEqual(expected);
+        });
+
+        // test
+        it('will return item with -10 durability and -1 enhancement', () => {
+
+            // Arrange
+            const expected = {
+                name: "broad sword",
+                durability: 90,
+                enhancement: 17
+            };
+
+            // Act
+            const item = {
+                name: "broad sword",
+                durability: 100,
+                enhancement: 18
+            };
+
+            // Assert
+            expect(fail(item)).toEqual(expected);
+        });
+    });
+
+    // test suite
+    describe('the repair function', () => {
+
+        // test
+        it('items durability should return to 100 after it is repaired', () => {
+
+            // Arrange
+            const expected = {
+                name: "broad sword",
+                durability: 100,
+                enhancement: 20
+            };
+
+            // Act
+            const item = {
+                name: "broad sword",
+                durability: 30,
+                enhancement: 20
```

Comment:
To make all of these items with the same name, you could use a helper function to shorten your typing/copy&paste time a little (and also reduce the time taken by any readers or people just learning your code-base):
```javascript
describe('the enhancer module', () => {
  const makeSword = (enhancement = 0, durability = 100) => ({
    name: 'broad sword',
    durability,
    enhancement
  });

  // ...

  describe('the repair function', () => {
    it('items durability should return to 100 after it is repaired', () => {
      const expected = makeSword(20);
      const item = makeSword(20, 30);

      expect(repair(item)).toEqual(expected);
    });
  });
// ...
```